src/store/Entity/Getters.ts

```diff
@@ -0,0 +1,47 @@
+import { GetterTree } from 'vuex';
+import Dictionary from '@/common/interfaces/Dictionary';
+import StateInterface from '@/store/Entity/Properties';
+
+export const getters: GetterTree<StateInterface, any> = {
```

Comment:
getters without (!) additional parameters are accessed like object properties, so IMO they should also be named as such e.g. `id` instead of `getId`, `labels` instead of `getLabels`. `getLabelByLanguage` is fine, though.